src/NHibernate/Persister/Entity/AbstractEntityPersister.cs

```diff
@@ -4230,14 +4230,7 @@ public virtual void AfterReassociate(object entity, ISessionImplementor session)
 				}
 			}
 
-			// check to see if it is in the second-level cache
```

Comment:
`session` argument is not used anymore. So, as we're talking about 6.0 it shall be removed.